src/charactersheet/viewmodels/dm/encounter_sections/treasure_section/components/magic-item.view.html

```diff
@@ -0,0 +1,30 @@
+<!-- ko with: entity.value -->
+<div class="h3">
+  <span data-bind="text: name"></span>&nbsp;
+  <small data-bind="text: rarity"></small>
+</div>
+
+<hr />
+<div class="row">
+  <div class="col-sm-4 col-xs-12">
```

Comment:
Aren't there a bunch of other fields missing from here?

Reply:
I believe that's all of them. The previous DM form had the same fields and the player tools are the same.